_layouts/default.html

```diff
@@ -9,6 +9,7 @@
 		<meta name="language" content="de"/>
 		<meta name="content-language" content="de"/>
 		<meta name="publisher" content="simplabs, Malthe Luda" />
+    <meta name="apple-itunes-app" content="app-id=918237891, affiliate-data=myAffiliateData, app-argument=myURL">
```

Comment:
Yes, the `affiliate-data` and `app-argument` are optional
https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html